app/controllers/recipe_ingredients_controller.rb

```diff
@@ -0,0 +1,10 @@
+class RecipeIngredientsController < ApplicationController
+  def update
+    redirect = Recipe.find(params[:id])
```

Comment:
I think I see what you mean here, but I think it's a little dangerous and potentially confusing calling a variable in a controller `redirect`, especially when you're only using it to create a RecipeIngredient.... which is not entirely correct.